pydoc.el

```diff
@@ -471,7 +477,7 @@ Adapted from `help-make-xrefs'."
 
 (defun pydoc-pip-version ()
   "Return a list of (major minor revision) for the pip version."
-  (let* ((output (shell-command-to-string "pip --version"))
+  (let* ((output (shell-command-to-string pydoc-pip-command))
```

Comment:
don't you need to change this variable name too?

Reply:
Oops! I force pushed some new changes with all the fixes.